src/wallet/rpcwallet.cpp

```diff
@@ -101,7 +101,11 @@ static void WalletTxToJSON(interfaces::Chain& chain, interfaces::Chain::Lock& lo
     {
         entry.pushKV("blockhash", wtx.hashBlock.GetHex());
         entry.pushKV("blockindex", wtx.nIndex);
-        entry.pushKV("blocktime", LookupBlockIndex(wtx.hashBlock)->GetBlockTime());
+        int64_t block_time;
+        if (!chain.findBlock(wtx.hashBlock, nullptr /* CBlock */, &block_time)) {
+            throw std::logic_error("invalid wallet transaction block hash");
```

Comment:
in commit c4b750f:

It is not immediately clear where this exceptions goes to. Crashes the node?

I'd prefer if we sticked to jsonrpc exceptions in the rpc module, so that broken rpc code would not crash the node.